README.md

```diff
@@ -1230,6 +1232,16 @@ versions can cause issues with the sandbox due to global state sharing. Temporal
 protobuf 4.x library unless you absolutely cannot at which point some proto libraries may have to be marked as
 [Passthrough Modules](#passthrough-modules).
 
+### Known Compatibility Issues
+
+Below are known compatibility issues with the Python SDK.
+
+#### gevent Patching
+
+When using `gevent.monkey.patch_all()`, asyncio event loops can get messed up, especially those using custom event loops
```

Comment:
Does it break the event loop in a predictable way? Just wondering if we could put the error here or in the github issue so people searching for the specific error find this.

Reply:
Probably, just need research. I admit I have not set up gevent and done all the testing. Can probably just raise an error in client creation if gevent has patched asyncio. I'm just being lazy atm in just adding docs. Given time, I will do full research around #59.